mmaction/datasets/rawframe_dataset.py

```diff
@@ -77,11 +76,11 @@ class RawframeDataset(BaseDataset):
         sample_by_class (bool): Sampling by class, should be set `True` when
             performing inter-class data balancing. Only compatible with
             `multi_class == False`. Only applies for training. Default: False.
-        power (float | None): We support sampling data with the probability
+        power (float): We support sampling data with the probability
```

Comment:
If DistributedPowerSampler is deleted, then what's the usefulness of power

Reply:
For now, I will set dynamic_length as False, so that the behaviour should be exactly the same as DistributedPowerSampler.